visma/io/parser.py

```diff
@@ -100,6 +100,76 @@ def resultStringCLI(equationTokens, operation, comments, solutionType, simul=Fal
     return finalSteps
 
 
+def resultMatrix_String(operation=None, operand1=None, operand2=None, nonMatrixResult=False, result=None):
```

Comment:
```suggestion
def resultMatrixString(operation=None, operand1=None, operand2=None, nonMatrixResult=False, result=None):
```
Since we are following camelCase throughout. Same in line 137.